focus_areas/code_development.md

```diff
@@ -26,7 +26,7 @@ Goal **Activity**:
 
 * Question **Changes**: How many changes are happening to the code base, during a certain time period? 
 
-  * Metric **Code_Changes_No**(Period): Number of changes to the code base (see [Code_Changes][../metrics/Code_Changes.md])
+  * Metric **Code_Changes_No**(Period): Number of changes to the code base (see [Code_Changes](../metrics/Code_Changes.md])
```

Comment:
line 29: The closing square bracket `]` after ".md" seems out of place

Reply:
Of course you're right. Thanks!